lib/kits/events/tracker.ts

```diff
@@ -116,7 +116,7 @@ class Tracker {
         return this.trackEvent(1, properties, version);
     }
 
-    trackPagedPublicationPageDisappeared(properties, version?: number) {
+    trackPagedPublicationPageOpened(properties, version?: number) {
```

Comment:
This is actually a breaking change for anyone who has manually been calling `eventTracker. trackPagedPublicationPageDisappeared()` directly instead of letting the `PagedPublicationViewer` do it for them - but I don't believe we have integrators matching that profile at all.